apps/etl/extraction/sources/idu/extract.py

```diff
@@ -0,0 +1,163 @@
+import json
+import logging
+
+import requests
+from django.conf import settings
+
+from apps.etl.extraction.sources.base.extract import Extraction
+from apps.etl.extraction.sources.base.utils import (
+    hash_file_content,
+    manage_duplicate_file_content,
+)
+from apps.etl.models import ExtractionData
+
+logger = logging.getLogger(__name__)
+
+
+class IDUExtraction(Extraction):
+    """
+    Handles data extraction from the IDU API for hazard data.
+    """
+
+    BASE_URL = "https://helix-tools-api.idmcdb.org/external-api/idus/last-180-days/"
```

Comment:
This link is also available at apps/etl/etl_tasks/idu.py
Can we have the link at one place only ?